src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -297,18 +318,39 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
         </template>
     </ModalCard>
 
+    <ModalCard v-else-if="activeStep === 'REFRESH_MOD_LIST'" key="REFRESH_MOD_LIST" :is-active="isOpen" :can-close="false">
+        <template v-slot:header>
+            <h2 class="modal-title">Refreshing mod list</h2>
+        </template>
+        <template v-slot:footer>
+            <div>
+                <p>
+                    At least some of the packages were not found. We're trying to fix this by refreshing the online mod list:</br>
+                    {{$store.state.tsMods.thunderstoreModListUpdateStatus}}
+                </p>
+            </div>
+        </template>
+    </ModalCard>
+
     <ModalCard v-else-if="activeStep === 'REVIEW_IMPORT'" key="REVIEW_IMPORT" :is-active="isOpen" @close-modal="closeModal">
         <template v-slot:header>
             <h2 class="modal-title">Packages to be installed</h2>
         </template>
         <template v-slot:body>
-            <OnlineModList :paged-mod-list="knownProfileMods" :read-only="true" />
+            <OnlineModList
```

Comment:
A bit unrelated to the changes, but I would probably change the order so that warning box is shown on top and the `OnlineModList` below it. The rationale is that for large profiles user might not scroll the list to the bottom and don't then see the warning.